### PR TITLE
Daniel Migault's editorial feedback

### PR DESCRIPTION
**Pull request by ounsworth**

### CODE REVIEW

**Comment by ounsworth** [comment]
NOT READY TO MERGE YET; I'm only halfway through processing Daniel's feedback.

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -118,10 +118,7 @@ The certificates typically contain one or more certification paths
 rooted in a device manufacture trust anchor and the leaf certificate being
 on the device in question; the latter is the Attestation Key that signs the Evidence statement.
 
-This document specifies two ATTRIBUTE/Attribute definitions. The first
-Attribute may be used to carry a set of certificates or public keys that
-may be required to validate signed Evidence. The second Attribute carries a
-structure that may be used to convey Evidence.
+This document specifies a CSR Attribute (or Extension for CRMF CSRs) for carrying evidence. An evidence statement in an existing format can be placed into an EvidenceStatement along with an OID to identify its type and optionally a hint to the Relying Party about how to verify it. A set of EvidenceStatements may be grouped together along with the set of CertificateAlternatives needed to validate them to form a EvidenceBundle. One or more EvidenceBundles may be placed into the id-aa-evidenceStatement CSR Attribute (or CFRM Extension).
```
Comment (hannestschofenig) [inline draft-ietf-lamps-csr-attestation.md:121]:
```suggestion
This document specifies a CSR Attribute (or Extension for Certificate Request Message Format (CRMF) CSRs) for carrying Evidence. Evidence can be placed into an EvidenceStatement along with an OID to identify its type and optionally a hint to the Relying Party about where to verify it. A set of EvidenceStatements may be grouped together along with the set of CertificateAlternatives needed to validate them to form a EvidenceBundle. One or more EvidenceBundles may be placed into the id-aa-evidenceStatement CSR Attribute (or CRFM Extension).
```

File: draft-ietf-lamps-csr-attestation.md [hunk 2]
```diff
@@ -196,7 +194,7 @@ or physically separated from the core CA functionality.
  .------------.               .----|----------.
  |            +-------------->|----'          | Compare Attestation
  |  Attester  |   Evidence    | Relying       | Result against
- |            |   in CSR      | Party (RA/CA) | policy
+ |   (HSM)    |   in CSR      | Party (RA/CA) | policy
```
Comment (henkbirkholz) [inline draft-ietf-lamps-csr-attestation.md:197]:
```suggestion
 |  (with HSM)    |   in CSR      | Party (RA/CA) | policy
```

Comment (henkbirkholz) [reply draft-ietf-lamps-csr-attestation.md:197]: The HSM is not the Attstester, just a part of one of the Attester's Attesting Environments

File: draft-ietf-lamps-csr-attestation.md (hunk 1 above)
Comment (ounsworth) [reply draft-ietf-lamps-csr-attestation.md]: I included this change manually.

**Pull request merged**